frontend/src/components/Pages/FrontPage/MissionOverview/ScheduleMissionDialog/SelectMissionsToScheduleDialog.tsx

```diff
@@ -51,10 +51,12 @@ export const SelectMissionsToScheduleDialog = ({ echoMissionsList, closeDialog }
         if (!selectedRobot) return
 
         selectedEchoMissions.forEach((mission: EchoMissionDefinition) => {
-            BackendAPICaller.postMission(mission.echoMissionId, selectedRobot.id, installationCode).catch(() => {
+            BackendAPICaller.postMission(mission.echoMissionId, selectedRobot.id, installationCode).catch((e) => {
                 setAlert(
                     AlertType.RequestFail,
-                    <FailedRequestAlertContent message={`Failed to schedule mission ${mission.name}`} />
+                    <FailedRequestAlertContent
+                        message={TranslateText('Failed to schedule mission') + ` '${mission.name}'. ${e.message}`}
+                    />
```

Comment:
There is translation on message before sending as input and inside FailedRequestAlertContent. Suggest removing translation on message inside FailedRequestAlertContent and instead set translation on all messages before sending as input. Perhaps also renaming message to translatedMessage